lib/src/compile.ts

```diff
@@ -12,14 +12,19 @@ import {Dispatcher} from './embedded-protocol/dispatcher';
 import {deprotifyException} from './embedded-protocol/utils';
 import {InboundMessage} from './vendor/embedded-protocol/embedded_sass_pb';
 
+type CompileResult = {
```

Comment:
```suggestion
export type CompileResult = {
```

Could you also export this from `lib/index.ts` so that it becomes part of the top-level API?